resources/views/annotations/show.blade.php

```diff
@@ -63,6 +63,7 @@
                 <button type="button" class="btn btn-default" v-on:click="dismissCrossOriginError">Continue at own risk</button>
             </div>
         </message-curtain>
+        <div v-show="showImage">
```

Comment:
You can add `v-show="!imageLoadingError"` directly to the attributes of the `<annotation-canvas>`. Then you don't need an additional wrapper.